apps/web/public/static/locales/en/common.json

```diff
@@ -2457,7 +2457,7 @@
   "create_account_password_hint": "You don't have an account password, create one by navigating to Security -> Password. Can't disconnect until an account password is created.",
   "disconnect_account": "Disconnect connected account",
   "disconnect_account_hint": "Disconnecting your connected account will change the way you log in. You will only be able to login to your account using email + password",
-  "cookie_consent_checkbox": "I consent to our privacy policy and cookie usage",
+  "cookie_consent_checkbox": "I consent to this website's privacy policy and cookie usage",
```

Comment:
What about this? `I consent to cal's privacy policy and cookie usage`

Reply:
I think, that would work too. Is `Cal` valid as a name? Because on the official website it says `Cal.com` everywhere. My thought was that using `Cal.com` would not be suitable for self-hosted instances which is why I went with `this website's`. What's your take on this?

Regarding locale files, I added the Spanish translation and I changed the translations for all locale files where `cookie_consent_checkbox` was present, except Hebrew (he) as I don't speak that language and have no means to verify if the translation would be valid. I could just copy the translation from a translation engine (e.g. Google translate). Do you want me to do that although I don't speak the language?

Reply:
Choose between this two- `By creating an account, you agree to our privacy policy and cookie usage` or
`By signing up, you agree to our privacy policy and cookie usage`